pages/release-notes.mdx

```diff
@@ -166,6 +175,86 @@ Memgraph (v2.11.1 and lower).
 
 </Callout>
 
+## Memgraph v2.21.0 and MAGE v1.21.0 - Nov 6, 2024
+
+{<h3> New features and improvements </h3>}
+
+- Added planner optimization to replace `point.distance` filter with an index
+  scan. Queries like `WITH point({x:0,y:0}) AS a MATCH (m:L1) WHERE
+  point.distance(m.prop, a) < 1 RETURN m;`, should run faster if there is the
+  point index created. If you are dealing with spatial data, there is a high
+  chance you need such indexes to run your queries faster.
+  [#2362](https://github.com/memgraph/memgraph/pull/2362)
+- Significantly improved schema retrieval performance in both TRANSACTIONAL and
+  ANALYTICAL setups. Feel free to use `--schema-info-enabled=True` to get
+  schema instantly by running the `SHOW SCHEMA INFO;` query.
+  [#2383](https://github.com/memgraph/memgraph/pull/2383)
+- Added query metadata when logging incoming query. It's possible now, e.g., to
```

Comment:
This can be updated with a how-to? https://memgraph.com/docs/database-management/logs#filtering-unnecessary-log-messages

Reply:
It can, but later? I'll resolve this comment here when merging to main.

Reply:
Sure. Reminder: https://github.com/memgraph/documentation/issues/1031